src/nvim/os/process.c

```diff
@@ -86,24 +86,16 @@ bool os_proc_tree_kill(int pid, int sig)
 }
 #else
 /// Kills process group where `pid` is the process group leader.
+/// TODO(vim): have an option to only kill the process, not the group?
```

Comment:
I'd say the comment is not worth having.

Reply:
Will remove it.